integrations/slack/package.json

```diff
@@ -6,7 +6,8 @@
     "build": "pnpm bp build",
     "deploy": "pnpm bp deploy",
     "type:check": "tsc --noEmit",
-    "test": "echo \"Tests not implemented yet.\""
+    "test": "echo \"Tests not implemented yet.\"",
+    "bp": "bp"
```

Comment:
c quoi ca

Reply:
I'm using globally installed bp cli so I don't have to do this anymore `npm i -g @botpress/cli`